src/test/java/nl/ordina/jobcrawler/scrapers/JobBirdScraperTest.java

```diff
@@ -63,224 +38,23 @@ public void init() {
-        parent1.appendChild(el1);
-        parent1.appendChild(el2);
-        Element parent2 = new Element("parent2");
-        parent2.appendChild(parent1);
-        parent2.appendChild(el3);
-
-        when(documentMock.select("span.page-link")).thenReturn(children);
-        int count = jobBirdScraperTestable.getTotalNumberOfPages(documentMock);
-        assertEquals(2, count);
-    }
-
-    /* set vacancy title using vacancy htm file */
-    @Test
-    void setVacancyTitle_HappyFlow() throws IOException {
-        Document doc = getDocFromUrl("JobBird/jobbird03_vacancy.htm");
-        String result = jobBirdScraperTestable.getVacancyTitle(doc);
-        assertEquals("Applications Engineering - Software Engineering Internship (Fall 2020)", result);
-    }
-
-
-    /*
-     *  If title cannot be found in the html a HTMLStructureException should be thrown
-     */
-    @Test
-    void setVacancyTitle_invalidPageStructure() throws IOException {
-        Document doc = getDocFromUrl("JobBird/jobbird03_vacancy_notitle.htm");
-        assertEquals("", jobBirdScraperTestable.getVacancyTitle(doc));
-    }
-
-    /*
-     *  Happy flow, location, hours and date exist in page
-     */
-    @Test
-    void setVacancySpecifics_happyFlow() throws IOException {
-        Document doc = getDocFromUrl("JobBird/jobbird04_vacancyspecifics.htm");
-        VacancyDTO vacancyDTO = VacancyDTO.builder().build();
-        vacancyDTO.setHours(jobBirdScraperTestable.retrieveWorkHours(doc.select("div.card-body").text()));
-        vacancyDTO.setLocationString(jobBirdScraperTestable.getLocation(doc));
-        vacancyDTO.setPostingDate(jobBirdScraperTestable.getPublishDate(doc));
-        assertEquals("Apeldoorn", vacancyDTO.getLocationString());
-        assertEquals(32, vacancyDTO.getHours());
-        LocalDateTime expectedDate =
-                LocalDateTime.parse("2020-05-30 00:00", DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm"));
-        assertEquals(expectedDate, vacancyDTO.getPostingDate());
-    }
-
-
-    /* Unhappy flow, either location, hours or date cannot be located, an empty string should be returned
-     */
-    @Test
-    void setVacancySpecifics_Missing() throws IOException {
-        Document doc = getDocFromUrl("JobBird/jobbird04_vacancyspecifics_missing.htm");
-        VacancyDTO vacancyDTO = VacancyDTO.builder().build();
-        vacancyDTO.setHours(jobBirdScraperTestable.retrieveWorkHours(doc.select("div.card-body").text()));
-        vacancyDTO.setLocationString(jobBirdScraperTestable.getLocation(doc));
-        vacancyDTO.setPostingDate(jobBirdScraperTestable.getPublishDate(doc));
-        assertEquals("", vacancyDTO.getLocationString());
-        assertNull(vacancyDTO.getHours());
+    void testGetVacancies() {
+        List<VacancyDTO> vacancyDTOList = jobBirdScraper.getVacancies();
+        assert(vacancyDTOList.get(0).getVacancyURL()).contains("www.jobbird.com");
+        assertNotNull(jobBirdScraper.getVacancies());
```

Comment:
In principe werkt de test prima. Denk echter dat deze test op deze wijze niet wenselijk is. Deze test is afhankelijk van een werkende internetconnectie en daardoor ook vrij langzaam. Runnen van deze test duurt +/- 25 seconden.
Denk dat we vrijdag tijdens de standup even moeten kijken hoe/wat. In de resources folder (/test/resources/JobBird) zitten de nodige html bestanden welke gebruikt zouden kunnen worden voor de test.

Indien we dit toch voor nu willen mergen, graag even regel 50 aanpassen naar `assertNotNull(vacancyDTOList);` Dat maakt de test in ieder geval al een stuk sneller :-)